### PR TITLE
OM-923 test flight changes

### PR DESCRIPTION
**Pull request by jgrund01**

### CODE REVIEW

File: FootlessParser.xcodeproj/FootlessParser_Info.plist [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <key>CFBundleSignature</key>
   <string>????</string>
   <key>CFBundleVersion</key>
-  <string>$(CURRENT_PROJECT_VERSION)</string>
+  <string>186</string>
```
Comment (kareman) [inline FootlessParser.xcodeproj/FootlessParser_Info.plist:21]: Where does 186 come from?

**Review by kareman — COMMENTED** [review]
Will other users be able to use this framework on versions earlier than 10.13?

File: FootlessParser.xcodeproj/FootlessParser_Info.plist (hunk 1 above)
Comment (jgrund01) [reply FootlessParser.xcodeproj/FootlessParser_Info.plist:21]: Apologies, this was completely my mistake!  This was done for some internal testing and was suppose to go against a custom fork!

**Comment by kareman** [comment]
No problem :-)

**Pull request closed without merging**